internal/handlers/common/aggregations/group.go

```diff
@@ -130,8 +130,8 @@ func newGroup(stage *types.Document) (Stage, error) {
 		if !ok {
 			return nil, commonerrors.NewCommandErrorMsgWithArgument(
 				commonerrors.ErrNotImplemented,
-				"unimplemented",
-				"$group",
+				fmt.Sprintf("$group accumulator %q is not implemented yet", operator),
+				operator+" (accumulator)",
```

Comment:
This will be the error message
```
test> db.test.aggregate({$group:{"_id": 1, "n": {$sum: 1}}})
MongoServerError: $group accumulator "$sum" is not implemented yet
```